dashboard/lib/policies/lti.rb

```diff
@@ -77,7 +77,14 @@ def self.get_account_type(roles)
   end
 
   def self.generate_auth_id(id_token)
-    "#{id_token[:iss]}|#{id_token[:aud]}|#{id_token[:sub]}"
+    case id_token[:aud]
+    when String
+      "#{id_token[:iss]}|#{id_token[:aud]}|#{id_token[:sub]}"
+    when Array
+      "#{id_token[:iss]}|#{id_token[:aud].first}|#{id_token[:sub]}"
```

Comment:
I'm reviewing the LTI spec and see what additional information they have on the Audience claim.
I will need to understand if just one client ID is expected, or if multiple client IDs can be in that list.
If I get a clear answer, I'll handle the different cases. Otherwise, I agree with having an output that monitors and collects data on unexpected values.